deploy-board/deploy_board/webapp/group_view.py

```diff
@@ -179,6 +180,11 @@ def update_launch_config(request, group_name):
             launchRequest["assignPublicIp"] = True
         else:
             launchRequest["assignPublicIp"] = False
+
+        if params["userData"].find("spiffe_id"):
```

Comment:
what if there was a spiffe_id in the userData and user did not change it but change other part of the userdata, we should still support this case.

Reply:
params is coming from request. I think this protection is good enough. 
The main purpose here is preventing user to update spiffe_id.
If data coming from user which is params does not contain spiffe_id, we are good.